libs/notebook/testdata/.ruff.toml

```diff
@@ -0,0 +1,2 @@
+[format]
+exclude = ["*.ipynb"]
```

Comment:
Let's not do this? `.ipynb` files are a part of the built-in templates and should be formatted.

Reply:
This exclude is scoped to the testdata folder. The testdata includes malformed files and ruff fails on those.